client/Packages/com.beamable.server/Editor/Usam/Editor/CodeService.cs

```diff
@@ -69,11 +69,12 @@ public async Promise Init()
 		{
 			if (EditorApplication.isPlayingOrWillChangePlaymode)
 				return;
+			
+			UsamLogger.ResetLogTimer();
+			
 			UsamLogger.Log("Running init");
-			_services = GetBeamServices();
-			UsamLogger.Log("Have services");
-			_storages = GetBeamStorages();
-			UsamLogger.Log("Have storages");
+			GetBeamServicePosts(out _services, out _storages);

```

Comment:
There are more places in our Unity SDK that could use this improvement. One example is in the `OnGeneratedSlnSolution` method we call both `GetBeamServices` and `GetBeamStorage` to go through all csproj files and reference them in Unity sln. We do use these methods in other places too, so it would be nice to check if they can use this

Reply:
Made a few more use cases of this method.